tests/test_publishing.py

```diff
@@ -169,24 +172,31 @@ def test_save_publishing_error_when_postoffice_raises_connection_error(
 @pytest.mark.django_db
 class TestBulkPublishing:
 
-    @patch('postoffice_django.publishing.requests.post')
     def test_bulk_messages_endpoint_called_when_calling_bulk_publish(
-            self, post_mock):
+            self, postoffice_valid_response):
+
+        responses.add(responses.POST,
+                      POSTOFFICE_PUBLISH_MESSAGE_URL,
+                      status=201,
+                      body=postoffice_valid_response,
+                      content_type='application/json')
+
         bulk_publish(
             topic='some_topic',
             payload=[{'key': 'key_1'}, {'key': 'key_2'}],
             hive='vlc1',
         )
+        request = responses.calls[0].request
+        breakpoint()
```

Comment:
I guess this was for testing! :)

Reply:
![](https://media.giphy.com/media/TJawtKM6OCKkvwCIqX/giphy.gif)